device-sdks/index.html

```diff
@@ -0,0 +1,2205 @@
+device.send_object_with_timestamp(&quot;com.example.GPS&quot;, &quot;/coords&quot;, coords, chrono::offset::Utc::now()).await?;</code></pre>
+
+        <h3
+          id="setting-and-unsetting-properties"
+          class="sdk-heading section-heading"
+        >
+          <a href="#setting-and-unsetting-properties" class="hover-link"
+            ><span class="icon-link" aria-hidden="true"></span
+          ></a>
+          Setting and Unsetting Properties
+        </h3>
+        <p class="sdk-paragraph">
+          <code class="code-inline">properties</code> represent a persistent,
+          stateful, synchronized state with no concept of history or
+          timestamping. From a programming point of view, setting and unsetting
+          properties of device-owned interface is rather similar to sending
+          messages on datastream interfaces.
+        </p>
+        <p class="sdk-paragraph">
+          Following examples show how to send a value that will be inserted into
+          the
+          <code class="code-inline">&quot;/sensor0/name&quot;</code> property
+          which is defined by
+          <code class="code-inline">&quot;/%{sensor_id}/name&quot;</code>
+          parametric endpoint (that is part of
+          <code class="code-inline"
+            >&quot;org.astarte-platform.genericsensors.AvailableSensors&quot;</code
+          >
+          device-owned properties interface).
+        </p>
+
+        <p class="sdk-paragraph">Rust:</p>
+        <pre
+          class="code-container"
+        ><code class="rust">/// set property (same as datastream)
+device.send(&quot;org.astarte-platform.genericsensors.AvailableSensors&quot;, &quot;/sensor0/name&quot;, &quot;foobar&quot;);
+
+/// unset property
+device.unset(&quot;org.astarte-platform.genericsensors.AvailableSensors&quot;, &quot;/sensor0/name&quot;);</code></pre>
+      </div>
+    </section>`;
+
+      var htmlContent = htmlCElement;
+      function populateData(id) {
+        var test = document.getElementById(id);
+        test.classList.add("active");
+        var list = listIds.filter((e) => e !== id);
+        list.forEach((e) => {
+          var elementById = document.getElementById(e);
+          elementById.classList.remove("active");
+        });
+
+        switch (id) {
+          case "C":
+            htmlContent = htmlCElement;
+            break;
+          case "C++":
+            htmlContent = htmlCPlusElement;
+            break;
+          case "Elixir":
+            htmlContent = htmlElixirElement;
+            break;
+          case "Go":
+            htmlContent = htmlGoElement;
+            break;
+          case "Java":
+            htmlContent = htmlJavaElement;
+            break;
+          case "Python":
+            htmlContent = htmlPythonElement;
+            break;
+          case "Rust":
+            htmlContent = htmlRustElement;
+            break;
+          default:
+            console.log("default");
```

Comment:
This looks like a leftover from debugging. Let's remove unneeded `console.log`

Reply:
Removed unneeded console.log